packages/ibm-products/src/components/FullPageError/FullPageError.mdx

```diff
@@ -0,0 +1,42 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
+import {
+  getStoryId,
+  CodesandboxLink,
+} from '../../global/js/utils/story-helper';
+import { FullPageError } from '.';
+
+# FullPageError
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+- [Component API](#component-api)
+
+## Overview
+
+The FullPageError component renders a full-page error message to inform users
```

Comment:
@matthewgallo @davidmenendez Have we traditionally included the same overview / page description from the design documentation here?

Reply:
Okay, we can use the same overview from the PAL website[^1] To be fair, we haven’t always been consistent in doing this, but I do think we should follow this approach going forward.

[^1]: https://pages.github.ibm.com/cdai-design/pal/patterns/full-page-errors/usage/